shared-ui/components/example-button/PrimaryButton.styles.ts

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+import { max } from '../../lib/responsive';
+import { colors } from '../../style/colors';
+
+const StyledPrimaryButton = styled.button`
+  color: ${colors.HEADER_FOOTER_BLUE};
+  background-color: ${colors.BUTTON_GREEN};
+  border-color: ${colors.HEADER_FOOTER_BLUE};
+  font-size: 1.0em;
+  background-color: ${colors.BUTTON_GREEN};
+  padding: 1em 1em;
```

Comment:
In the design the primary button on desktop looks taller than the secondary button - let's replicate that here instead of making the padding the same as the secondary button.

On mobile the sizing looks about the same on the design so that part looks fine! Just adjust the desktop sizing to be more similar to the design:)

Reply:
This might be due to font size and the button expanding to fit the larger font.

Reply:
Ah this is true - when we change the button font to use the font changes from the other PR we can make those final touches then!